src/main.py

```diff
@@ -1,14 +1,22 @@
-import json
-import logging as log
-from copy import copy
-from os import getenv
+# -------------------------> Dependencies
+
+# Setup python logging
+import logging
+log = logging.getLogger(__name__)
 
+# Import libraries
+from copy import copy
 import discord
 from discord.ext import commands
 from discord.ext.commands.errors import ExtensionAlreadyLoaded, ExtensionNotFound, CheckFailure
-from dotenv import load_dotenv
+import json
+import logging as log
```

Comment:
this `log`  will break since its overwritten via an import later

Reply:
Yeah pretty sure we don't need that `log` import. And any other code before imports is a no-no imo

Reply:
whoops